docs/create_allocator_service.md

```diff
@@ -0,0 +1,335 @@
+# Quickstart Create an Allocator Service
+
+This quickstart describes one way to implement a [Service](https://kubernetes.io/docs/concepts/services-networking/service/) which will allocate a Game Server on demand using the [Agones API](https://godoc.org/agones.dev/agones/pkg/client/clientset/versioned/typed/stable/v1alpha1).  This allows a game client or matchmaker service to call the allocator service in order to allocate a Game Server and retrieve its JSON encoded GameServerStatus.
+
+This type of service could be used by a game client to connect directly to a dedicated Game Server, as part of a larger system, such as a matchmaker service, or in conjunction with a database to enable level transition data.
+
+## Objectives
+- Create a secure allocator service
+- Deploy the service to GKE
+- Allocate a Game Server from a Fleet using the service
+
+## Prerequisites
+1. [Docker](https://www.docker.com/get-started/)
+2. Agones installed on GKE, running a simple-udp fleet
+3. kubectl properly configured
+4. A clone or fork of the [Agones repo](https://github.com/GoogleCloudPlatform/agones)
```

Comment:
What do we need the a clone/fork for? Since we are using the build pattern in Docker, we don't actually need this, no?

Reply:
Actually ignore me - I'm removing my objection to this - I think it's fine.